racket/repl.rkt

```diff
@@ -128,8 +128,8 @@
 (define listener
   (tcp-listen 0  ;choose port dynamically, a.k.a. "ephemeral" port
               64
-              #f ;reuse? not recommended for ephemeral ports
-              "127.0.0.1"))
+              #t ;reuse? not recommended for ephemeral ports
```

Comment:
Regarding the security concern: I think it might be fine if the remote back end is started using a newly added command-line argument such as `--listen-to` (defaulting to 127.0.0.1) and that value is used here. Unfortunately I guess one flaw to supplying the "what is my IP?" value automatically is the usual complication of NATs. So not yet sure best thing to do here.

Reply:
That would be nice, we need a way to find out the IP from elisp